Backend/lean_fridge_flask.py

```diff
@@ -0,0 +1,71 @@
+from flask import Flask, render_template, url_for, request, redirect, flash
+app = Flask(__name__)
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from lean_fridge_setup import Base, Stock
+
+engine = create_engine('sqlite:///lean_fridge_app.db')
+Base.metadata.bind = engine
+
+DBSession = sessionmaker(bind = engine)
+session = DBSession()
+
+@app.route('/')
+@app.route('/stock/')
+def index():
+    stock = session.query(Stock).all()
+    return render_template('index.html', stock = stock)
+
+@app.route('/stock/<int:item_id>/')
```

Comment:
this page works if i go to the URL directly but you didn't link it anywhere from the main page, right?